Classes/Core/Runtime/FormRuntime.php

```diff
@@ -191,7 +182,7 @@ protected function initializeFormStateFromRequest()
             $this->formState = new FormState();
         } else {
             $serializedFormState = $this->hashService->validateAndStripHmac($serializedFormStateWithHmac);
-            $this->formState = unserialize(base64_decode($serializedFormState));
+            $this->formState = unserialize(base64_decode($serializedFormState), [FormState::class]);
```

Comment:
```suggestion
            $this->formState = unserialize(base64_decode($serializedFormState), ['allowed_classes' => [FormState::class]]);
```

Reply:
Thanks! I knew it felt strange, but couldn't pinpoint that feeling…